modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -241,6 +255,7 @@ def _wrap_partitions(self, partitions):
     def apply(
         self,
         func,
+        *args,
```

Comment:
It feels more natural that way, but due to the way it's currently implemented, `*args` must be here since it must match the behavior of `PandasOnRayDataframePartition ` here: https://github.com/modin-project/modin/blob/master/modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py#L90-L101, which accepts arbitrary args first, then keyword args.